es-core/src/InputManager.cpp

```diff
@@ -101,7 +101,14 @@ void InputManager::addJoystickByDeviceIndex(int id)
 	// set up the prevAxisValues
 	int numAxes = SDL_JoystickNumAxes(joy);
 	mPrevAxisValues[joyId] = new int[numAxes];
-	std::fill(mPrevAxisValues[joyId], mPrevAxisValues[joyId] + numAxes, 0); //initialize array to 0
+	mInitAxisValues[joyId] = new int[numAxes];
+
+	int axis;
+	for (int i = 0; i< numAxes; i++) {
+		axis = SDL_JoystickGetAxis(joy, i);
+		mInitAxisValues[joyId][i] = axis;
+		mPrevAxisValues[joyId][i] = axis;
```

Comment:
I just want to make sure: previously we were initializing to 0, and now we're initializing to...? The axis value at the time this runs?

Will this add problems with non-problematic triggers?

Just checking, as I'm not extremely knowledgeable about the input handling here.

Reply:
Yes, it's initializing to the initial axis value reported by the device. As to whether it can cause conflicts, I'm not certain... but in the original fork, someone proposed a similar change, ostensibly to prevent unwanted scrolling: https://github.com/Aloshi/EmulationStation/pull/587

Reply:
Looks good. Thanks.